### PR TITLE
UE-252: Add Lora Google Font

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# NOT READY FOR REVIEW

# Summary
- TL;DR - what's this PR for?

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: vpue_undergrad_subtheme.libraries.yml [hunk 1]
```diff
@@ -100,3 +100,9 @@ allpages:
     theme:
       dist/css/theme.css: {}
       dist/css/print.css: { media: print }
+
+fonts:
+  css:
+    theme:
+      'fonts.googleapis.com/css2?family=Lora:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&display=swap': { type: external, minified: true }
```
Comment (katrialesser) [inline vpue_undergrad_subtheme.libraries.yml]: @jdwjdwjdw can you try this line starting with // like I did here:
https://github.com/SU-SWS/saa_victoria_subtheme/commit/170e3f52b59cc78a8a5adfc0f3f15676c0483988
Or try with http:// for testing to get it to work initially. The // format will allow the site to load it with either http or https depending what the site requests or uses by default, I believe.
I would try http:// version on that line 107 as a first debugging step and see if it works.

**Review by katrialesser — CHANGES_REQUESTED** [review]
I made a suggestion to get that font library to work.

File: vpue_undergrad_subtheme.libraries.yml (hunk 1 above)
Comment (jdwjdwjdw) [reply vpue_undergrad_subtheme.libraries.yml]: Thanks @katrialesser! Looks like that did the trick 😄

**Review by buttonwillowsix — CHANGES_REQUESTED** [review]
@jdwjdwjdw this looks like it does more than the font. Is this addressing some other subtheme weirdness, like what we saw with the search box?  Let me know and let's get this merged today.

**Comment by jdwjdwjdw** [comment]
> @jdwjdwjdw this looks like it does more than the font. Is this addressing some other subtheme weirdness, like what we saw with the search box? Let me know and let's get this merged today.

ahh sorry for the delay @buttonwillowsix !

I'm not sure why all those dist files changed, but the font is the only main thing I've addressed here. I did also rename the incorrect vpue_undergrad_subtheme.yml file to vpue_undergrad_subtheme.libraries.yml, but I don't think that addresses any of the search weirdness.

**Pull request merged**